pyro/contrib/forecast/evaluate.py

```diff
@@ -200,12 +200,19 @@ def forecaster_options_fn(*args, **kwargs):
             "num_samples": num_samples,
             "train_walltime": train_walltime,
             "test_walltime": test_walltime,
+            "params": {},
         }
         results.append(result)
         for name, fn in metrics.items():
             result[name] = fn(pred, truth)
-            if isinstance(result[name], (int, float)):
-                logger.debug("{} = {}".format(name, result[name]))
+        for name, value in pyro.get_param_store().items():
+            if value.numel() == 1:
```

Comment:
make the numel boundary toggleable so that users have an easy way to inspect params? e.g. they can even choose inf

Reply:
I'm trying to avoid logging any `Tensor`s, instead logging only Python objects. One reason for this design choice is that a single big tensor renders the entire data structure unreadable (visually, at a terminal). One option that would still satisfy this design constraint is to log scalar statistics of each non-scalar tensor, e.g. min, max, median, mean. That seems like a reasonable feature for a follow-up PR.